### PR TITLE
Sundials/ML 6.0.0p0

### PR DESCRIPTION
**Pull request by tbrk**
Update package for latest release of Sundials/ML. User code that works with the the 5.8.0p0 release should continue working without modification on this new release.

### CODE REVIEW

File: packages/sundialsml/sundialsml.6.0.0p0/opam [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+install: [
+  [make "install-findlib"]
+  [make "install-doc"] {with-doc}
+]
+remove: [
+    ["ocamlfind" "remove" "sundialsml"]
+    ["rm" "-rf" "%{doc}%/sundialsml"]
+]
+depends: [
+    "ocaml" {>= "4.03.0"}
```
Comment (mseri) [inline packages/sundialsml/sundialsml.6.0.0p0/opam:36]:
```suggestion
    "ocaml" {>= "4.08.0"}
```

the option module was introduced in 4.08. If you need to remain compatible with older releases you can use stdcompat to backport option to the old compilers

**Comment by mseri** [comment]
Looks fine. Let me separately add upper bounds to zelus

**Comment by tbrk** [comment]
Thanks @mseri. I should have tested compilation under 4.03 before pushing! Sorry about that.

**Comment by mseri** [comment]
No problem. The CI is there for this :)

File: packages/zelus/zelus.2.1/opam [hunk 2]
```diff
@@ -14,6 +14,7 @@ depends: [
   "menhir"
```
Comment (mseri) [inline packages/zelus/zelus.2.1/opam:14]:
```suggestion
  "menhir" {>= "20180523"}
```
Uses --infer-write-query

**Comment by tbrk** [comment]
That looks reasonable to me. Zelus itself has been [updated](https://github.com/INRIA/zelus/pull/17) but not yet rereleased.

**Comment by tbrk** [comment]
That looks reasonable to me. Zelus itself has been [updated](https://github.com/INRIA/zelus/pull/17) but not yet rereleased.

**Comment by tbrk** [comment]
Thanks @mseri!

**Pull request merged**